co-authors-plus.php

```diff
@@ -1469,6 +1472,15 @@ public function filter_jetpack_open_graph_tags( $og_tags, $image_dimensions ) {
 		// Send back the updated Open Graph Tags
 		return apply_filters( 'coauthors_open_graph_tags', $og_tags );
 	}
+
+	/**
+	 * Callback to clear the cache on post save and post delete.
+	 *
+	 * @param $post_id The Post ID.
+	 */
+	public function clear_cache( $post_id ) {
+		wp_cache_delete( 'coauthors_post_' . $post_id );
```

Comment:
I believe that this call to `wp_cache_delete` will need to pass the cache group `'co-authors-plus'` as its second argument, in order to ensure that it is cleared properly - since the cache item was added in the `'co-authors-plus'` cache group.